internal/cmd/run/cypress.go

```diff
@@ -54,6 +55,11 @@ func NewCypressCmd() *cobra.Command {
 					Username:   credentials.Get().Username,
 					ConfigFile: gFlags.cfgFilePath,
 				})
+				bt.Report(err, map[string]interface{}{
```

Comment:
bt docs says this should be `map[string]string`

Reply:
Any reason to not make this an explicit type?

Reply:
The parameter is `map[string]interface{}` for this function. And I tried to pass `map[string]string` here. It's not allowed.